SqlInstall.ps1

```diff
@@ -0,0 +1,433 @@
+﻿function Install-DbaInstance {
```

Comment:
Please rename the file as well to Install-DbaInstance.ps1 in the functions directory

Reply:
Can i change filename and location without messing up this thread?

Reply:
great q, im unsure. you can make all of the changes you'd like and then rename, then ping me and ill give it a fresh look